src/views/shared/FooterMFA.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import hbs from 'handlebars-inline-precompile';
+import FooterSignout from './FooterSignout';
+
+export default FooterSignout.extend({
+  template: hbs(
+    '\
+      {{#if hasFactorPageCustomLink}}\
+        <a href="{{factorPageCustomLinkHref}}" data-se="factor-page-custom-link" \
+          class="link js-factor-page-custom-link" \
+          rel="noopener noreferrer" target="_blank">\
+          {{factorPageCustomLinkText}}\
+        </a>\
+      {{/if}}\
+      {{#if showLink}}\
```

Comment:
we should have each link in a different line. Reason is that in other languages, if we have them side by side, there might be some overlap or inconsistencies

Reply:
@vincentvoong-okta ^ any thoughts ?

Reply:
@mauriciocastillosilva-okta @haishengwu-okta 
that sounds good to me. it will look like this: 
![image](https://user-images.githubusercontent.com/14948016/107253102-d4e56800-69ea-11eb-97ff-1a4c137c490a.png)

Reply:
Done